CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # unreleased
 
+- Allow env vars to override config files, but not command-line arguments, see #1152, #1281, and #2381 (@aaronkollasch)
```

Comment:
Can we please move this to the "Features" subsection below (merge or rebase with master first, there are some entries in there by now)? I think we should also highlight that this is a "breaking change", right? Just in case someone relied on the previous behavior (config takes precedence over env). Maybe:
```suggestion
- Breaking change: Environment variables can now override config file settings (but command-line arguments still have the highest precedence), see #1152, #1281, and #2381 (@aaronkollasch)
```